assets/js/admin-settings/rule.jsx

```diff
@@ -0,0 +1,145 @@
+/* global wp, wpLocalizedData */
+import {
+  Button,
+  SelectControl,
+  TextControl,
+  ToggleControl,
+} from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+import React, { useEffect, useState } from 'react';
+import { useSelect } from '@wordpress/data';
+import { ruleCard } from './styles';
+
+
+const Rule = ({
+  field,
+  loading,
```

Comment:
`loading` and `saving` can likely be combined into one prop called `busy`, since it seems like all usages of them in this component are in the form of `loading || saving`, and then the prop could be set by `busy={loading || saving}`